libtelemetry/src/main/java/com/mapbox/android/telemetry/VisionObjectDetectionEvent.java

```diff
@@ -0,0 +1,212 @@
+  private Double object_lon;
+  @SerializedName("vehicle_lat")
+  private Double vehicle_lat;
+  @SerializedName("vehicle_lon")
+  private Double vehicle_lon;
+  @SerializedName("class")
+  private String clazz;
+  @SerializedName("sign_value")
+  private String sign_value;
+  @SerializedName("object_size_width")
+  private Double object_size_width;
+  @SerializedName("object_size_height")
+  private Double object_size_height;
+  @SerializedName("object_pos_height")
+  private Double object_pos_height;
+  @SerializedName("distance_from_camera")
+  private Double distance_from_camera;
+
+  VisionObjectDetectionEvent(String created) {
+    this.event = VIS_OBJECT_DETECTION;
+    this.created = created;
+    this.object_lat = null;
+    this.object_lon = null;
+    this.vehicle_lat = null;
+    this.vehicle_lon = null;
+    this.clazz = null;
+    this.sign_value = null;
+    this.object_size_height = null;
+    this.object_size_width = null;
+    this.object_pos_height = null;
+    this.distance_from_camera = null;
+  }
+
+  public String getEvent() {
+    return event;
+  }
+
+  public String getCreated() {
+    return created;
+  }
+
+  public double getObjectLatitude() {
+    return object_lat;
+  }
+
+  public void setObjectLatitude(double latitude) {
+    this.object_lat = latitude;
+  }
+
+  public double getObjectLongitude() {
+    return object_lon;
+  }
+
+  public void setObjectLongitude(double longitude) {
+    this.object_lon = longitude;
+  }
+
+  public double getVehicleLatitude() {
+    return vehicle_lat;
+  }
+
+  public void getVehicleLatitude(double latitude) {
+    this.vehicle_lat = latitude;
+  }
+
+  public double getVehicleLongitude() {
+    return vehicle_lon;
+  }
+
+  public void setVehicleLongitude(double longitude) {
+    this.vehicle_lon = longitude;
+  }
+
+  public String getClazz() {
+    return clazz;
+  }
+
+  public void setClazz(String clazz) {
+    this.clazz = clazz;
+  }
+
+  public String getSignValue() {
+    return sign_value;
+  }
+
+  public void setSignValue(String sign_value) {
+    this.sign_value = sign_value;
+  }
+
+  public double getObjectSizeWidth() {
+    return object_size_width;
+  }
+
+  public void setObjectSizeWidth(double object_size_width) {
+    this.object_size_width = object_size_width;
+  }
+
+  public double getObjectSizeHeight() {
+    return object_size_height;
+  }
+
+  public void setObjectSizeHeight(double object_size_height) {
+    this.object_size_height = object_size_height;
+  }
+
+  public double getObjectPositionHeight() {
+    return object_pos_height;
+  }
+
+  public void setObjectPositionHeight(double object_pos_height) {
+    this.object_pos_height = object_pos_height;
+  }
+
+  public double getDistanceFromCamera() {
+    return distance_from_camera;
+  }
+
+  public void setDistanceFromCamera(double distance_from_camera) {
+    this.distance_from_camera = distance_from_camera;
+  }
+
+  @Override
+  Event.Type obtainType() {
+    return Event.Type.VIS_GENERAL;
```

Comment:
VIS_OBJECT_DETECTION ?

Reply:
@yunikkk good catch! although i think the types are in general useless :)